contracts/characters.sol

```diff
@@ -140,6 +140,10 @@ contract Characters is Initializable, ERC721Upgradeable, AccessControlUpgradeabl
         return success ? secondsLeft : 0;
     }
 
+    function burn(uint256 tokenId) public restricted {
```

Comment:
AFAIK, character-burning may be something ordinary users can do too in the future? I think this function maybe should be renamed to clarify it's for more admin-y purposes. I would like it to also have a dedicated, more specific, role instead of `GAME_ADMIN` that the `restricted` modifier enforces, and maybe also check that `msg.sender` is the owner as well.

The use of `GAME_ADMIN` for everything and anything has been pretty rampant through the entire contract codebase and I'd like to change that.